File: ADO/suredata2/Backup/suredata1/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Data.SqlClient;

namespace suredata1
{
    public partial class Form2 : Form
    {
        SqlConnection cn;
        SqlDataAdapter da;
        SqlCommandBuilder cmb;
        DataSet ds;
        DataTable dt;
        public Form2()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            DataRow dr = dt.NewRow();

            dr[0] = textBox1.Text;
            dr[1] = textBox2.Text;
            dr[2] = textBox3.Text;

            dt.Rows.Add(dr);

            da.Update(ds, "emp");
            MessageBox.Show("Row Created");
        }

        private void Form2_Load(object sender, EventArgs e)
        {
            cn = new SqlConnection
               ("Data Source=SERVER;Initial Catalog=college;User ID=sa");
            da = new SqlDataAdapter("select * from emp", cn);
            cmb = new SqlCommandBuilder(da);

            ds = new DataSet();

            da.Fill(ds, "emp");

            dt = ds.Tables["emp"];
        }

        private void button2_Click(object sender, EventArgs e)
        {
            dataGridView1.ClearSelection();
            dataGridView1.DataSource = ds.Tables[0];
        }

        private void button3_Click(object sender, EventArgs e)
        {
            DataRow[] dr;
            dr = dt.Select("eid=" + textBox1.Text);

            if (dr.Length > 0)
            {
                dr[0].Delete();
                da.Update(ds, "emp");
                MessageBox.Show("Record Deleted");
            }
            else
                MessageBox.Show("Record Not Found");
           
        }

        

       
    }
}

File: ADO/ado_vb/ado_vb/Form2.vb
Imports System.Data.SqlClient
Imports System.Data

Public Class Form2
    Dim c As SqlConnection
    Dim cmb As SqlCommandBuilder
    Dim da As SqlDataAdapter

    Dim ds As New DataSet()
    Dim dt As DataTable

    Private Sub Form2_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        c = New SqlConnection("Data Source=SUNETHRA\ARASU;Initial Catalog=ado;Integrated Security=True")
        da = New SqlDataAdapter("select * from emp", c)
        cmb = New SqlCommandBuilder(da)
        da.Fill(ds, "emp")

        dt = ds.Tables("emp")
        DataGridView1.ClearSelection()
        DataGridView1.DataSource = ds.Tables("emp")
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim dr As DataRow
        dr = dt.NewRow()
        dr(0) = TextBox1.Text
        dr(1) = TextBox2.Text
        dr(2) = TextBox3.Text


        dt.Rows.Add(dr)

        da.Update(ds, "emp")

        DataGridView1.DataSource = ds.Tables("emp")
        MessageBox.Show("Row Created")
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        DataGridView1.ClearSelection()
        DataGridView1.DataSource = ds.Tables("emp")

    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        Dim dr() As DataRow

        dr = dt.Select("eid=" + TextBox1.Text)

        If (dr.Length > 0) Then
            '   dr(0).Delete()
            dt.Rows.Remove(dr(0))
            da.Update(ds, "emp")
            MessageBox.Show("Record Deleted")
        Else
            MessageBox.Show("Record Not Found")
        End If

    End Sub

    Private Sub Button4_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button4.Click
        Dim dr() As DataRow

        dr = dt.Select("eid=" + TextBox1.Text)


        If (dr.Length > 0) Then

            Dim ch = MsgBox("Are u want to update", MsgBoxStyle.YesNo)
            If (ch = MsgBoxResult.Yes) Then

                dr(0)(1) = TextBox2.Text
                dr(0)(2) = TextBox3.Text
                da.Update(ds, "emp")
                MessageBox.Show("Record Updated")

            End If

        Else
            MessageBox.Show("Record Not Found")
        End If
           

    End Sub
End Class